src/blocks/subscribe/edit.js

```diff
@@ -125,6 +157,33 @@ export default function SubscribeEdit( {
 						/>
 					) }
 				</PanelBody>
+				<PanelBody
+					title={ __( 'Styles', 'newspack-newsletters' ) }
+					initialOpen={ false }
```

Comment:
Nit: This forces the panel to be closed when the editor refreshes. Omitting the attribute lets the open state persist from the last state (so if you open the panel then refresh the editor, it stays open, and vice versa). This is the default behavior for most Gutenberg sidebar panels, so it feels like we should follow the convention.

Reply:
Oh good call! This has been updated in [d1b0f75](https://github.com/Automattic/newspack-newsletters/pull/1283/commits/d1b0f7528db4ff627a4b6a0796adc30de3dcc378).